animation_nodes/math/euler.pyx

```diff
@@ -0,0 +1,7 @@
+from . number cimport lerpFloat
+
+cdef void mixEul3(Euler3* target, Euler3* a, Euler3* b, float factor):
```

Comment:
Why the `3` here?

Reply:
I added because `Euler3` has `3`. Should I remove `3`?

Reply:
In this case, you may leave it as is.

Reply:
Okay.